src/rpc/rawtransaction.cpp

```diff
@@ -284,8 +284,13 @@ UniValue gettxoutproof(const JSONRPCRequest& request)
         pblockindex = mapBlockIndex[hashBlock];
     } else {
         CCoins coins;
-        if (pcoinsTip->GetCoins(oneTxid, coins) && coins.nHeight > 0 && coins.nHeight <= chainActive.Height())
```

Comment:
seems like this is the last txid, not the first in the list? Little confused on current behavior.